bridges/centralized-ethereum/Cargo.toml

```diff
@@ -9,14 +9,15 @@ actix = { version = "0.11.1", default-features = false }
 async-jsonrpc-client = { git = "https://github.com/witnet/async-jsonrpc-client", features = ["tcp"], branch = "fix-tcp-leak" }
 ctrlc = "3.1.3"
 env_logger = "0.7.1"
+ethabi = "14.0.0"
```

Comment:
I think this line is not needed, you can revert all the `use ethabi::` into `use web3::ethabi`.